packages/vitest/src/node/reporters/json.ts

```diff
@@ -88,7 +88,7 @@ export class JsonReporter implements Reporter {
     this.start = Date.now()
   }
 
-  protected async logTasks(files: File[], coverageMap?: CoverageMap | null) {
+  protected async logTasks(files: File[], coverageMap?: CoverageMap | null): Promise<JsonTestResults> {
```

Comment:
`logTasks` no longer logs tasks. Should we change a name?

Reply:
yeah... it generates the logs, it just leaves writing up to the caller.

Reply:
Change the name then